lib/matplotlib/axes/_axes.py

```diff
@@ -1849,6 +1849,8 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
             If `edge`, aligns bars by their left edges (for vertical bars) and
             by their bottom edges (for horizontal bars). If `center`, interpret
```

Comment:
Since we are fixing the docstring here, shouldn't `edge` and `center` have regular quotes and not backticks, since they are string values, not arguments?